src/auth/auth.service.ts

```diff
@@ -33,4 +38,28 @@ export class AuthService {
 
     return this.userService.create(userDto);
   }
+
+  async login(user: JwtUser) {
+    const token = this.jwtService.sign(user);
+
+    return {
+      accessToken: token,
+      statusCode: HttpStatus.CREATED,
+    };
+  }
+
+  async validateUser(username: string, pass: string) {
```

Comment:
better not use a abbreviation to name a variable. I would **recommend** to name it as full as it is "password"

Reply:
Revised.
[refactor: revised recommend in auth.service.ts](https://github.com/Cophr/backend/pull/10/commits/a7106295d99711d88471d9a81f22865281b49e3b)